docs/dcp2_system_design.rst

```diff
@@ -960,221 +960,91 @@ Project-level matrices
-provided via TDR snapshots. The MVP snapshot will permanently expose the
-interim artifacts, while a later release snapshot will expose the
-permanent replacements.
-
-For DCP/2 MVP, no post-processing is performed on contributor-generated
-matrices. Users will be able to download them in exactly the same file
-format as they were provided by the contributor. For example, if the
-contributor provided a ZIP archive of multiple matrix files, the user
-will need to download that archive and extract it locally.
-
-The interim solution stores the contributor-generated matrices as data
-files in the TDR and describes them with a `file
-descriptor <Generic file metadata_>`_, a ``supplementary_file`` entity
-and a ``supplementary_link`` entry in a ``links.json``.
-There will be one subgraph (and therefore one ``links.json``
-document) per HCA project. If the contributor provided multiple matrices
-stratified by, say, species and organ, the per-project subgraph will
-contain a ``supplementary_file`` and a
-``supplementary_link`` for each such matrix file.
-
-.. image:: media/project-cgm.png
-   :align: center
+``Schema Changes for Projects (managed access)``
+
+Managed Access projects have additional data we want to be able to capture. We previously did not have the fields to capture this data in our metadata schema. We will therefore create two new optional fields.
+
+- DbGAP Accession Field
 
-The Data Browser back end (Azul) indexes the
-``supplementary_file`` entities for these contributor-generated
-matrices as it would any other ``supplementary_file`` entity and
-exposes them as entries in its ``/index/files`` response. This enables
-the Data Browser to list them on its *Files* tab and make them
-discoverable and downloadable like any other data file.
+- EGA Accession Field
 
-While this is no different to how other supplementary files are handled,
-Azul provides special treatment of supplementary files that describe
-contributor-generated matrices (CGMs): each hit in Azul's
-``/index/project`` response contains a listing of all CGMs so that those
-matrices can be exposed on the Data Browser's Projects tab, both the
-list view and the project details view.
+DUO (Data Use Ontology) codes will not be included at this time.
 
-The set of properties the metadata schema currently allows for in
-``supplementary_file`` entities is limited, so the challenge is
-how to encode matrix stratification. The schema defines the following
-properties:
 
--  ``supplementary_file.provenance.submitter_id``
 
--  ``supplementary_file.file_core.format``
+``Schema changes for CGMs (dcp3)``
 
--  ``supplementary_file.file_core.content_description``
+For release 3 and onwards, we intend to store matrices provided by data contributors (as opposed to the matrices generated by the DCP) in the informationally rich metadata schema. The CGMs will be described as ‘analysis_file’ entities, similar to DCP-generated matrices, and will be linked via a process to:
```

Comment:
Don't use future tense. It will be confusing once the future arrives. Use present tense.